backstop/tests/carousel.html

```diff
@@ -0,0 +1,270 @@
+            data-slide-id="2"
+            role="group"
+            aria-roledescription="slide"
+            aria-label="3 of 3"
+            aria-hidden="true"
+          >
+            <div
+              class="iui-carousel-slider-item-image"
+              style="background-image: url('../assets/image-test.jpg');"
+            ></div>
+            <div class="iui-carousel-slider-item-title">Bentley System headquarters</div>
+          </li>
+
+        </ol>
+
+        <nav class="iui-carousel-navigation">
+          <button
+            class="iui-button iui-borderless iui-small iui-carousel-navigation-button iui-carousel-navigation-previous"
+            type="button"
+            aria-controls="myCarousel-items"
+            aria-label="Previous slide"
+          >
+            <svg-chevron-left
+              class="iui-button-icon"
+              aria-hidden="true"
+            ></svg-chevron-left>
+          </button>
+
+          <button
+            class="iui-button iui-borderless iui-small"
+            aria-label="Pause animation"
+            data-action="pause"
+          >
+            <svg-pause
+              class="iui-button-icon"
+              aria-hidden="true"
+            ></svg-pause>
+          </button>
+
+          <ol class="iui-carousel-navigation-dots">
+            <li>
+              <button
+                class="iui-active iui-carousel-navigation-dot"
+                data-slide="0"
+                aria-label="Slide 1 (Current slide)"
+              ></button>
+            </li>
+            <li>
+              <button
+                class="iui-carousel-navigation-dot"
+                data-slide="1"
+                aria-label="Slide 2"
+                id="test-1"
+              ></button>
+            </li>
+            <li>
+              <button
+                class="iui-carousel-navigation-dot"
+                data-slide="2"
+                aria-label="Slide 3"
+                id="test-2"
+              ></button>
+            </li>
+          </ol>
+
+          <!-- Replicates the width of the play/pause button to center the dots -->
+          <div style="width: 28px;"></div>
+
+          <button
+            class="iui-button iui-borderless iui-small iui-carousel-navigation-button iui-carousel-navigation-next"
+            type="button"
+            aria-controls="myCarousel-items"
+            aria-label="Next slide"
+          >
+            <svg-chevron-right
+              class="iui-button-icon"
+              aria-hidden="true"
+            ></svg-chevron-right>
+          </button>
+        </nav>
+      </div>
+
+    </section>
+
+    <script>
+      document.querySelectorAll('.iui-carousel').forEach((carousel) => {
+        const slides = [...carousel.querySelectorAll('.iui-carousel-slider-item')];
+        const [prev, next] = [...carousel.querySelectorAll('.iui-carousel-navigation-button')];
+
+        const updateSlideClasses = (current) => {
```

Comment:
Maybe do the same with the dots do? Not that much of the code (basically copy paste) and we could have fully functional carousel.

Reply:
Dots now update to show which slide is active.  I haven't tried making the dots clickable yet.

Reply:
Made the dots clickable in [`39ea6ce`](https://github.com/iTwin/iTwinUI/pull/377/commits/39ea6ce28f838987cc0bedf9718244926f178d31)